src/app/shared/date.service.ts

```diff
@@ -73,4 +73,22 @@ export class DateService {
       return 'n/a';
     }
   }
+
+  getISO8601FormatFromDate(timestamp: Date): string {
+    if (timestamp) {
+      const tsAsDate = new Date(timestamp);
```

Comment:
Needlessly creating another date

Reply:
here Date seems to just be an alias for a number, the argument has no `toISOString` method until initialized as a new Date. Combined these two methods into one.